e2e/test_people_page.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import yaml
+import os
+
+import pytest
+
+from bok_choy.web_app_test import WebAppTest
+from pages.people_page import PeoplePage
+
+test_shard = os.getenv('TEST_SHARD')
+
+class TestConfiguration(WebAppTest):
+    
+    def setUp(self):
+        super(TestConfiguration, self).setUp()
+        config_path = os.getenv('CONFIG_PATH')
+        try:
+            yaml_contents = open(
+                    "{}/user_config.yml".format(config_path), 'r'
+                    ).read()
+        except IOError:
+            pass
```

Comment:
This might be tricky to debug if something unexpected does go wrong with loading the file.

Reply:
fixed